src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -29,62 +29,89 @@
+$multiday_classes = [ 'tribe-events-calendar-month__multiday-event' ];
 
 // @todo @fe move class configuration to template tag
 
 if ( $event->featured ) {
 	$classes[] = 'tribe-events-calendar-month__multiday-event--featured';
 }
 
+$display_tooltip = ! empty( $event->excerpt ) || ! empty( $event->cost ) || $event->thumbnail->exists;
+
 // If it starts today and this week, let's add the left border and set the width.
 if ( $should_display ) {
 
 	/*
 	 * The "duration" here is how many days the event will take this week, not in total.
 	 * The two values might be the same but they will differ for events that last more than one week.
 	 */
-	$classes[] = 'tribe-events-calendar-month__multiday-event--width-' . $event->this_week_duration;
+	$multiday_classes[] = 'tribe-events-calendar-month__multiday-event--width-' . $event->this_week_duration;
+	$event_classes[]    = 'tribe-events-calendar-month__multiday-event--display';
 
 	// If it ends this week, let's add the start class (left border).
 	if ( $event->starts_this_week ) {
-		$classes[] = 'tribe-events-calendar-month__multiday-event--start';
+		$multiday_classes[] = 'tribe-events-calendar-month__multiday-event--start';
 	}
 
 	// If it ends this week, let's add the end class (right border).
 	if ( $event->ends_this_week ) {
-		$classes[] = 'tribe-events-calendar-month__multiday-event--end';
+		$multiday_classes[] = 'tribe-events-calendar-month__multiday-event--end';
 	}
 
 	if ( $event->dates->end->format( 'Y-m-d' ) < $today_date ) {
-		$classes[] = 'tribe-events-calendar-month__multiday-event--past';
+		$multiday_classes[] = 'tribe-events-calendar-month__multiday-event--past';
 	}
 
-
-} else {
-	$classes[] = 'tribe-events-calendar-month__multiday-event--hidden';
 }
 
 ?>
 <div class="tribe-events-calendar-month__multiday-event-wrapper">
-	<article <?php tribe_classes( $classes ); ?> data-event-id="<?php echo esc_attr( $event->ID ); ?>">
-		<time
-			datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
-			class="tribe-common-a11y-visual-hide"
-		>
-			<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
-		</time>
-		<a href="<?php echo esc_url( $event->permalink ) ?>" class="tribe-events-calendar-month__multiday-event-inner">
-			<?php if ( $event->featured ) : ?>
-				<em
-					class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
-					aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-					title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-				></em>
-			<?php endif; ?>
-			<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
-				<?php echo get_the_title( $event ); ?>
-			</h3>
-		</a>
+	<article <?php tribe_classes( $event_classes ); ?> data-event-id="<?php echo esc_attr( $event->ID ); ?>">
+		<div class="tribe-events-calendar-month__multiday-event--hidden">
```

Comment:
You have this as `__multiday-event` class, but also you use the `__multiday-event` class above in `$multiday_classes`, which is used below. These are 2 different elements, they should have different classes. Also, this wrapper will always be for the hidden portion of the event, so no need for the `--hidden` modifier.